memo/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+
+# Note: This crate must be built using do.sh
+
+[package]
+name = "spl-memo"
+version = "0.1.0"
+description = "Solana Program Library Memo"
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2018"
+
+[dependencies]
+num-derive = "0.2"
+num-traits = "0.2"
+solana-sdk = { version = "=1.2.0", default-features = false, features=["program"] }
```

Comment:
Why the hard `=`?

Reply:
The precedent here (Token) is to pin to a particular version of the solana-sdk. We still need to work out release/deployment processes for the programs in this repo. But I do think the `=` is helpful from a readability perspective, assuming what appears here is meant to represent what is published to the chain.

Reply:
We'll need to add release tags any time we're publishing to the chain.  Definitely can't assume what's in the master branch is meaningful. Can't really assume what's outside the Cargo.lock is too meaningful either.  `=` deps are bad for downstream consumers.